src/number_theory/bernoulli_polynomials.lean

```diff
@@ -33,7 +33,7 @@ Bernoulli polynomials are defined using `bernoulli`, the Bernoulli numbers.
 ## TODO
 
 - `bernoulli_poly_eval_one_neg` : $$ B_n(1 - x) = (-1)^n*B_n(x) $$
-- ``bernoulli_poly_eval_one` : Follows as a consequence of `bernoulli_poly_eval_one_neg`.
+- `bernoulli_poly_eval_one` : Follows as a consequence of `bernoulli_poly_eval_one_neg`.
```

Comment:
```suggestion
- `bernoulli_poly_eval_one` : follows as a consequence of `bernoulli_poly_eval_one_neg`.
```

Reply:
I am not so sure about this. If wee start with a lower case we should also change for example the ones in https://leanprover-community.github.io/mathlib_docs/number_theory/bernoulli_polynomials.html#main-theorems ?

Reply:
Looking at the [guidelines](https://leanprover-community.github.io/contribute/style.html) we should use lowercase characters. Feel free to make the relevant modifications :)

Reply:
OK, I see that in the guidelines sometimes lowercase letters are used and sometime uppercase... I *think* lowercase is the preferred one.

Reply:
Ok, thank you for that, i made the changes at https://github.com/leanprover-community/mathlib/pull/11788/commits/0ce3f21d11f46b91479dc06c9a5575f192def189